invenio_access/__init__.py

```diff
@@ -261,10 +262,10 @@
 
 >>> with app.test_client() as c:
 ...     c.get('/')
-<Response streamed [403 FORBIDDEN]>
+<WrapperTestResponse streamed [403 FORBIDDEN]>
```

Comment:
Werkzeug has renamed it [here](https://github.com/pallets/werkzeug/blob/76de1b82c7ef9c692cf3731a4de53d7b213f3bf7/src/werkzeug/test.py#L873)